gateway/sds_gateway/api_methods/views/capture_endpoints.py

```diff
@@ -588,15 +589,7 @@ def _infer_index_name(
         return request_data
 
     # Populate index_name based on capture type
-    match capture_type:
-        case CaptureType.DigitalRF:
-            request_data["index_name"] = f"captures-{CaptureType.DigitalRF}"
-        case CaptureType.RadioHound:
-            request_data["index_name"] = f"captures-{CaptureType.RadioHound}"
-        case _:
-            msg = f"Invalid capture type: {capture_type}"
-            log.error(msg)
-            raise ValueError(msg)
+    request_data["index_name"] = infer_index_name(capture_type)
 
     return request_data
```

Comment:
nit: we can probably remove this _handle_capture_type function, as it's not doing much anymore